ygen/testdata/proto/proto-test-e.uncompressed.enums.formatted-txt

```diff
@@ -0,0 +1,34 @@
+// openconfig.enums is generated by codegen-tests as a protobuf
+// representation of a YANG schema.
+//
+// Input schema modules:
+//  - testdata/proto/proto-test-e.yang
+// Include paths:
```

Comment:
Suppress this line if empty, or indicate empty list []?

Reply:
Done.